charts/enterprise-metrics/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 1.7.2
+version: 1.7.3
```

Comment:
nit: I feel like it should be a minor version bump since we are introducing a new feature to the chart.